to_delist.txt

```diff
@@ -0,0 +1,6 @@
+# will be replaced by UNAL Ancizar Sans and UNAL Ancizar Serif (upcoming PRs)
+https://fonts.google.com/specimen/Ancizar+Sans
```

Comment:
Let's wait on these Ancizar delistings in the to_delist until we have the new families in production.